deploy/gcloud/Dockerfile.consumer

```diff
@@ -62,6 +62,10 @@ EXPOSE 8080
 
 ENV PATH=$PATH:.
 
+# This env var is used by click-to-deploy deployer, right now needs to be
+# manually synced with C2D_RELEASE in the other Docker images for AMP Packager
```

Comment:
Link to the click-to-deploy repo from somewhere (maybe here).

Reply:
Done. Also added in the renewer.